server/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # dev build
-FROM ghcr.io/immich-app/base-server-dev:20240507@sha256:ca3b8f7ca4ef72cb191b97d2715bfa0e3decdf39e666c5020536e41cf14cee1e as dev
+FROM ghcr.io/immich-app/base-server-dev:main as dev
```

Comment:
This change

Reply:
Oh, this was to use the base image with the Vulkan dependencies. There hasn't been a release since then so I just used main.